rss2017/process/pamsCode.js

```diff
@@ -0,0 +1 @@
+console.log('Process function executed');
```

Comment:
I was using this file as an example of how the process function .template files work. I can remove it.